code/modules/projectiles/magazines/pistols.dm

```diff
@@ -157,17 +157,23 @@
 
 /obj/item/ammo_magazine/pistol/heavy/super //Commander's variant
 	name = "\improper Heavy Desert Eagle magazine (.50)"
-	desc = "A heavy variant of Desert Eagle magazines specially-tuned to fit a high-ranking officer. This variant is optimized for anti-armor."
+	desc = "Seven rounds of devastatingly powerful 50-caliber destruction."
 	gun_type = /obj/item/weapon/gun/pistol/heavy/co
 	default_ammo = /datum/ammo/bullet/pistol/heavy/super
 	icon_state = "deagleS"
 
 /obj/item/ammo_magazine/pistol/heavy/super/highimpact
 	name = "\improper High Impact Desert Eagle magazine (.50)"
-	desc = "A heavy variant of Desert Eagle magazines specially-tuned to fit a high-ranking officer. This variant is optimized for anti-personnel. Don't fire it at anyone you'd like to stay alive."
+	desc = "Seven rounds of devastatingly powerful 50-caliber destruction. The bullets are tipped with a synthesized osmium and lead alloy to stagger absolutely anything they hit. Point away from anything you value."
 	default_ammo = /datum/ammo/bullet/pistol/heavy/super/highimpact
 	icon_state = "deagleE"
 
+/obj/item/ammo_magazine/pistol/heavy/super/highimpact/ap
+	name = "\improper High Impact Armor Piercing Desert Eagle magazine (.50)"
```

Comment:
`name = "\improper High Impact Armor-Piercing Mateba speed loader (.454)"`
pick one style for armour-piercing/armor piercing hyphenation